models/position_test.go

```diff
@@ -0,0 +1,65 @@
+package models_test
+
+import (
+	"fmt"
+
+	"go-crawler-challenge/models"
+	. "go-crawler-challenge/tests"
+	. "go-crawler-challenge/tests/fixtures"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Position", func() {
+	AfterEach(func() {
+		TruncateTable("position")
+	})
+
+	Describe("#GetAllPosition", func() {
+		Context("given an existing position", func() {
+			It("returns a position", func() {
+				existUser := FabricatePosition("nonAds", "#search .g .yuRUbf > a", "normal")
```

Comment:
Why is the variable named `existUser` 🙃 Did you mean to do:

```suggestion
				_ := FabricatePosition("nonAds", "#search .g .yuRUbf > a", "normal")
```